src/main/java/seedu/tp/commands/ResetCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.tp.commands;
+
+import seedu.tp.exceptions.HistoryFlashcardException;
+import seedu.tp.flashcard.FlashcardList;
+import seedu.tp.ui.Ui;
+
+public class ResetCommand extends Command {
```

Comment:
Consider using a more specific name like `ResetReviewedCommand` to indicate we're only resetting the reviewed status? Using `ResetCommand` might seem a bit misleading given that we are not resetting all fields in the flashcards :)